### PR TITLE
Update description

### PR DESCRIPTION
**Pull request by get-itips**
closes https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/issues/8891

aligning name and description of the option to updated setting https://docs.microsoft.com/en-us/microsoftteams/meeting-policies-participants-and-guests#let-anonymous-people-join-a-meeting

@MSFTTracyP please review

### CODE REVIEW

File: Teams/teams-security-guide.md [hunk 1]
```diff
@@ -287,7 +287,7 @@ Meeting organizers control whether participants can join a meeting without waiti
 The defaults are:
 
 - *People in my Organization* - Everyone external to the organization will wait in the lobby until admitted.
-- *People from my organization and trusted organizations* - Authenticated users and external users from Teams and Skype for Business domains that are in the external access allowlist can bypass the lobby. All other users will wait in the lobby until admitted.
+- *People in my organization, trusted organizations, and guests* - Authenticated users within the organization, including guest users and the users from trusted organizations, join the meeting directly without waiting in the lobby. Anonymous users wait in the lobby.
```
Comment (JohanFreelancer9) [inline Teams/teams-security-guide.md:290]:
```suggestion
- *People in my organization, trusted organizations, and guests* - Authenticated users within the organization, including guest users and users from trusted organizations, join the meeting directly without waiting in the lobby. Anonymous users wait in the lobby.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@MSFTTracyP PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by get-itips** [comment]
@MSFTTracyP please help me with the review

**Comment by get-itips** [comment]
@tiburd Tina, can you help me here? Thanks

**Comment by tiburd** [comment]
reviewed + confirmed

**Pull request merged**